### PR TITLE
Meowstrogen (juice that makes you meow :3)

### PR DESCRIPTION
**Pull request by KyuPolaris**
## About the PR

Added a chemical extracted from Cat Ears and Hairballs that acts similarly to the "Juice that makes you weh" but instead of going weh, you go meow :3

## Why / Balance

Another silly chemical, balanced around Cat Ear rarity and limitations of hairballs

## Technical details

YML Edits

## Media
Content Client_8vWODyLqIa

https://github.com/DeltaV-Station/Delta-v/assets/64847293/ef64f02b-81c6-409a-ba18-b6d10751dc5f

https://github.com/DeltaV-Station/Delta-v/assets/64847293/2b5b61d4-a829-4e19-b56d-a1c8065d9a7f

https://github.com/DeltaV-Station/Delta-v/assets/64847293/558fadf7-21c3-41dc-8b0c-64bbaffac359

https://github.com/DeltaV-Station/Delta-v/assets/64847293/7f0c5800-ea49-4d36-858e-85b6c44f279b

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

n/a

**Changelog**

🆑 
- add: The essence of cats and felinids (Meowstrogen) has been added

### CODE REVIEW

File: Resources/Prototypes/Entities/Clothing/Head/misc.yml
Comment (DangerRevolution) [inline Resources/Prototypes/Entities/Clothing/Head/misc.yml:1]: This needs to be in the Delta-V folder

File: Resources/Prototypes/Nyanotrasen/Entities/Objects/Specific/Species/felinid.yml [hunk 1]
```diff
@@ -15,6 +15,8 @@
         reagents:
         - ReagentId: Protein
           Quantity: 2
+        - ReagentId: Meowstrogen
```
Comment (DangerRevolution) [inline Resources/Prototypes/Nyanotrasen/Entities/Objects/Specific/Species/felinid.yml]: Not our code so this needs to have Delta-V comments.
```suggestion
        - ReagentId: Meowstrogen # Delta-V : Asda Meowstrogen
```

File: Resources/Prototypes/Nyanotrasen/Entities/Objects/Specific/Species/felinid.yml [hunk 2]
```diff
@@ -15,6 +15,8 @@
...
+          Quantity: 2
```
Comment (DangerRevolution) [inline Resources/Prototypes/Nyanotrasen/Entities/Objects/Specific/Species/felinid.yml]: As above:
```suggestion
          Quantity: 2 # Delta-V : Adds Meowstrogen
```

File: Resources/Prototypes/Entities/Clothing/Head/misc.yml
Comment (KyuPolaris) [reply Resources/Prototypes/Entities/Clothing/Head/misc.yml:1]: This is the base cat ears item, how would you want me to move it to the Delta folder? I am a bit confused

File: Resources/Prototypes/Nyanotrasen/Entities/Objects/Specific/Species/felinid.yml [hunk 3]
```diff
@@ -15,6 +15,8 @@
         reagents:
         - ReagentId: Protein
           Quantity: 2
+        - ReagentId: Meowstrogen # Delta-V : Asda Meowstrogen
```
Comment (DangerRevolution) [inline Resources/Prototypes/Nyanotrasen/Entities/Objects/Specific/Species/felinid.yml]:
```suggestion
        - ReagentId: Meowstrogen # Delta-V : Adds Meowstrogen
```

File: Resources/Prototypes/Entities/Clothing/Head/misc.yml
Comment (DangerRevolution) [reply Resources/Prototypes/Entities/Clothing/Head/misc.yml:1]: My bad; was very early in the day. This just needs Delta-V tags from start to bottom :)

Comment (KyuPolaris) [reply Resources/Prototypes/Entities/Clothing/Head/misc.yml:1]: All good, I'll fix that now. I did my best to do the tags but it was also pretty early in the morning when I did it. (it came to be in a dream)

**Comment by DEATHB4DEFEAT** [comment]
I don't like this. People already meow too much on their own and polymorphing is annoying and bad.

**Comment by Axionyxx** [comment]
I agree with Death, I don't like this either.

**Pull request closed without merging**